### DISCUSSION TITLE
Meu Amigo Yoshi

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#301] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/meu-amigo-yoshi/

**Comment by maryon-telvanni** [comment]
Vai a má fama do Mario seja por causa dessa imagem:



![null](<https://i.postimg.cc/R0s4cHRN/3459c799890937d3c18e16ce9fc419af.jpg>)
